project/Game.py

```diff
@@ -416,22 +421,22 @@ def test_answer_question_incorrect_no_time(self):
         # IT IS RECCOMENDED TO HAVE THESE TESTS BE LAST IN THE SUITE
 
     def test_answer_question_time_penalty(self):
-        self.team.clueTime = datetime.datetime.now()
+        self.team.clueTime = datetime.timedelta(hours=datetime.datetime.now().hour,minutes=datetime.datetime.now().minute,seconds=datetime.datetime.now().second)
         time.sleep(16)
         self.assertTrue(self.game.answer_question(self.team,"three disks"),"Returned False after Correct Answer!")
         self.assertEqual(self.team.points,230,"Points did not increment correctly")
         self.assertEqual(self.team.currentLandmark,2,"Landmark Index Did not Properly increment")
         self.assertEqual(len(self.team.timelog),3,"Time Log Did Not recieve a new entry")
-        self.assertEqual(self.team.timelog[2],self.team.clueTime+datetime.time(00,00,16),"Time Log Did Not Save The Correct Time") #This may not work properly
+        self.assertEqual(self.team.timelog[2],self.team.clueTime+datetime.timedelta(hours=0,minutes=00,seconds=16),"Time Log Did Not Save The Correct Time") #This may not work properly
 
     def test_answer_question_time_penalty2(self):
-        self.team.clueTime = datetime.datetime.now()
+        self.team.clueTime = datetime.timedelta(hours=datetime.datetime.now().hour,minutes=datetime.datetime.now().minute,seconds=datetime.datetime.now().second)
         time.sleep(31)
```

Comment:
If we factor out the calculation of datetime.now() like I did in my code review we no longer need these problematic sleeps. We can also then add tests for crossing day barriers (game takes multiple days).